### ISSUE TITLE
Enhance App Icon configuration

### ISSUE THREAD

**Issue by sgobotta** [Cambalab/ns-vue-radio#103]
It would be nice to have an easy way to configure the application icon from a config file or an image.

Listening to proposals.

Depends on #102

**Comment by estefi-prieto** [comment]
`tns resources generate icons <./image>`

Generates all icons for Android and iOS platforms and places the generated images in the correct directories under App_Resources/<platform> directory.

**Comment by sgobotta** [comment]
Can you organise that information in the README generated in #102?

Please, under the title Assets write down a subtitle App Icons generation and organise your instructions.

**Comment by estefi-prieto** [comment]
I'm working on this
